EventsExpress/Controllers/UserController.cs

```diff
@@ -53,9 +52,9 @@ IEmailService emailService
         /// <response code="200">Return IEnumerable UserManageDto models</response>
         /// <response code="400">Return failed</response>
         [HttpGet("[action]")]
-        public IActionResult SearchUsers([FromQuery]UsersFilterViewModel filter)
+        public IActionResult SearchUsers([FromQuery] UsersFilterViewModel filter)
         {
-            filter.PageSize = 6;     
+            filter.PageSize = 6;
```

Comment:
Is there a reason to have the `PageSize` be set to 6? If no, please remove this assignment.